### ISSUE TITLE
Waiting on some features

### ISSUE THREAD

**Issue by aarongerig** [w-vision/Wvision#21]
## On hold
- [x] Add static assets to `manifest.json` (waiting on [#175](https://github.com/symfony/webpack-encore/issues/175) and this [PR](https://github.com/symfony/webpack-encore/pull/221), maybe even this [PR](https://github.com/symfony/webpack-encore/pull/409/files))
- [ ] Configure StyleLint `.stylelintrc.js` (wait for [#56](https://github.com/airbnb/css/pull/56) to be merged)
